### PR TITLE
Small fix to migrate to pytorch 0.4.0

### PR DESCRIPTION
**Pull request by Kelvinson**
Hi, thanks you @kentsommer for this great implementation. I just did a small fix to migrate to 0.4.0.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -25,15 +24,13 @@ def train(net, trainloader, config, criterion, optimizer, use_GPU):
...
-            # Send Tensors to GPU if available
-            if use_GPU:
-                X = X.cuda()
-                S1 = S1.cuda()
-                S2 = S2.cuda()
-                labels = labels.cuda()
-            # Wrap to autograd.Variable
-            X, S1 = Variable(X), Variable(S1)
-            S2, labels = Variable(S2), Variable(labels)
+            # automaticlly select device, device agnostic
```
Comment (kentsommer) [inline train.py]: Please capitalize the first letter of the comment.

**Review by kentsommer — COMMENTED** [review]
Thanks for your contribution. Looks good, two minor nits

File: train.py (hunk 1 above)
Comment (Kelvinson) [reply train.py]: Hi @kentsommer , thanks for you reply, I solved at commit: [b18247](https://github.com/Kelvinson/pytorch-value-iteration-networks/commit/b1824f7a361efde199a0fc39a323767f08cf3b34) ,can you merge it now? Thank you and wish you a happy new year!

**Comment by kentsommer** [comment]
Thanks for the contribution @Kelvinson! Happy New Year :)

**Pull request merged**